Casks/k/kindle-create.rb

```diff
@@ -0,0 +1,28 @@
+cask "kindle-create" do
+  version "1.93.0.0"
+  sha256 :no_check
+
+  url "https://d2bzeorukaqrvt.cloudfront.net/KindleCreateInstaller.pkg",
+      verified: "https://d2bzeorukaqrvt.cloudfront.net/"
```

Comment:
```suggestion
      verified: "d2bzeorukaqrvt.cloudfront.net/"
```

The `verified` parameter should not have a URL scheme.